### DISCUSSION TITLE
【1364780】街霸六（仅本体

### DISCUSSION THREAD

**Discussion by yuzhouqishi111** [pjy612/SteamManifestCache#108] [Share]
appid:1364780
keys:1364781;cfec39716b5a5cae49b52b5c0e01e6db93356ee3df5a0d87590fd811ab6d06e1
内测码：E2572C74C908DC96·
邮箱：283204040@qq.com

**Comment by pjy612** [comment]
查收

**Reply by yuzhouqishi111** [reply]
收到啦
